joblib/test/test_numpy_pickle.py

```diff
@@ -278,9 +359,9 @@ def test_compressed_pickle_dump_and_load():
     # or smaller than cache_size)
     for cache_size in [0, 1e9]:
```

Comment:
You could remove the loop over cache_size here, since it doesn't have any effect.